
Well - fixed stupid mkinstalldirs postprocessing that fails for us ...

  - Rene Rebe <rene@exactcode.de>

--- gentoo-0.11.54/po/Makefile.in.in.vanilla	2005-03-23 16:44:40.324269336 +0100
+++ gentoo-0.11.54/po/Makefile.in.in	2005-03-23 16:44:50.926657528 +0100
@@ -27,7 +27,7 @@
 INSTALL = @INSTALL@
 INSTALL_DATA = @INSTALL_DATA@
 MKINSTALLDIRS = @MKINSTALLDIRS@
-mkinstalldirs = $(SHELL) `case "$(MKINSTALLDIRS)" in /*) echo "$(MKINSTALLDIRS)" ;; *) echo "$(top_builddir)/$(MKINSTALLDIRS)" ;; esac`
+mkinstalldirs = $(MKINSTALLDIRS)
 
 GMSGFMT = @GMSGFMT@
 MSGFMT = @MSGFMT@
